### ISSUE TITLE
Feature importance

### ISSUE THREAD

**Issue by arBest** [havakv/pycox#89]
Hello @havakv,

Firstly, great work for the community!

I think it would be very useful to understand important predictive features by the model and specifically was looking to see if can obtain the feature importance calculated using permutation importance for ex. as sksurv which is compatible with ELI [provides](https://scikit-survival.readthedocs.io/en/latest/user_guide/random-survival-forest.html?highlight=feature%20importance#Permutation-based-Feature-Importance) or if there are other routes to do so?

Greatly appreciated and thank you for your help!

**Comment by havakv** [comment]
Thank you for the kind words!
I agree that it is very useful to get a measure of feature importance. I don't think the `PermutationImportance` of `eli5` will work out of the box, but there is a guide on how to approach this issues with the `eli5.permutation_importance.get_score_importances()` here https://eli5.readthedocs.io/en/latest/blackbox/permutation_importance.html#model-inspection 

Maybe try that out and tell us if that seems to work?

**Comment by arBest** [comment]
Thanks @havakv! That's it.

Something like this would work. It's always great to have a second pair of eyes :) 

```
from eli5.permutation_importance import get_score_importances

def model_score(X, y):
    surv = y[0].interpolate(15).predict_surv_df(X.astype('float32'))
    get_target = lambda df: (df['os_time'], df['os_event'])
    durations_val, events_val = get_target(y[1])
    ev = EvalSurv(surv, durations_val, events_val, censor_surv='km')
    c_ind = ev.concordance_td('antolini')
    return c_ind

# pass trained model (estimator)
base_score, score_decreases = get_score_importances(model_score, test_x, (estimator, test_y), n_iter=5)
feature_importances = np.mean(score_decreases, axis=0)
```

**Comment by havakv** [comment]
Great that you were able to to make it work!
